dictdiffer/__init__.py

```diff
@@ -283,6 +285,9 @@ def patch(diff_result, destination, in_place=False):
                      Setting ``in_place=True`` means that patch will apply
                      the changes directly to and return the destination
                      structure.
+    :param allow_missing_keys: By default, trying to remove a key will raise
```

Comment:
Missing the word ”missing”. 😄 Here and in the other docstring.

Reply:
hehehe, you're right, updated!